README.md

```diff
@@ -372,13 +372,9 @@ easy to fork and contribute any changes back upstream.
          ~~~ zsh
          echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.zprofile
          echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.zprofile
-         echo 'eval "$(pyenv init --path)"' >> ~/.zprofile
-         
-         echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.profile
```

Comment:
As per the ZSH documentation, the '.profile' file is not used in this shell. It would be used only in special cases where we could expect that the more experienced user will know if they need to add the content in that file. Having this as default configuration for ZSH would be just adding messy files to the system.